src/main/resources/templates/beverage/copy.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="https://thymeleaf.org/">
+<head th:replace="fragments :: head"></head>
+<body class="text-center">
+<nav th:replace="fragments :: navigation"></nav>
+<img class = "sam" src = "https://docs.google.com/drawings/d/e/2PACX-1vRn_7yHWBEvtlOdRRo_qu2kcUVZoXeRpWpd1B2fBwq9NdHbjOq64m902PXtDWkBbaqqXkeuYL-FTQdi/pub?w=960&h=720">
+<div class="container-fluid">
+    <form method="post" th:action="@{/beverages/copy}">
+        <input type="hidden" th:value="${beverage.id}" name="beverageId">
+        <div th:replace="fragments :: beverage-form-group"></div>
+        <input type="submit" value="Copy" th:formaction="@{/beverages/copy}" class="btn btn-info">
+    </form>
+</div>
+<footer th:replace="fragments :: footer"></footer>
+<div th:replace="fragments :: bootstrap-javascript"></div>
+</body>
+</html>
```

Comment:
Nothing to change here, just commenting on this really elegant and succinct solution!